.gitignore

```diff
@@ -7,6 +7,9 @@ __pycache__/
 *.py[cod]
 .mypy_cache/
 
+# This file is allowed specifically because it is a required fixture.
+!tests/settings/basic/fixture_bad_pyc.pyc
```

Comment:
Can we please instead write this file in tests? It is highly likely to be removed by someone (me 😆), because I have workflows that remove `.pyc` files sometimes.

Reply:
Something like `f = open('tests/settings/basic/fixture_bad_pyc.pyc', 'w'); f.write('contents')`

Reply:
This should be fixed in bb77da8.

Reply:
That makes sense, addressed in bb77da8.